src/lfn.c

```diff
@@ -18,6 +18,14 @@
 
 BOOL IsFATName(LPTSTR pName);
 
+#ifndef IO_REPARSE_TAG_MOUNT_POINT

```

Comment:
Why are these defined here?  The values are in elsewhere (e.g., wfdirrd.c) and not defined there.

Reply:
No good reason.  I pushed a different commit to remove them.

(The real answer is that my tree is targeting 2000, and symlink in particular was added in Vista, so setting WINVER to anything less than that causes these to not be defined.)